impl_core/src/main/java/io/opencensus/implcore/internal/DaemonThreadFactory.java

```diff
@@ -38,14 +36,9 @@ public DaemonThreadFactory(String threadPrefix) {
 
   @Override
   public Thread newThread(Runnable r) {
-    Thread thread = threadFactory.newThread(r);
-    // AppEngine runtimes have constraints on thread renaming.
-    try {
-      thread.setName(threadPrefix + threadIdGen.getAndIncrement());
-      thread.setDaemon(true);
-    } catch (SecurityException e) {
```

Comment:
Is it safe to remove this try-catch clause?

Reply:
The constraints on thread renaming is from AppEngine threads. Since we are not using them it should be okay. But let me put it back as it doesn't hurt to have try-catch clause.